app/src/main/java/com/matmoongi/MainActivity.kt

```diff
@@ -4,19 +4,21 @@ import android.os.Bundle
 import androidx.activity.ComponentActivity
 import androidx.activity.compose.setContent
 import androidx.activity.viewModels
-import androidx.compose.material3.MaterialTheme
-import com.matmoongi.screens.LoginScreen
+import androidx.compose.material3.ExperimentalMaterial3Api
+import com.matmoongi.screens.SearchScreen
+import com.matmoongi.theme.MatmoongiTheme
 import com.matmoongi.viewmodels.FavoritesViewModel
 import com.matmoongi.viewmodels.SearchViewModel
 
+@ExperimentalMaterial3Api
 class MainActivity : ComponentActivity() {
-    val searchViewModel: SearchViewModel by viewModels()
-    val favoritesViewModel: FavoritesViewModel by viewModels()
+    private val searchViewModel: SearchViewModel by viewModels()
+    private val favoritesViewModel: FavoritesViewModel by viewModels()
```

Comment:
```suggestion
    private val favoritesViewModel: FavoritesViewModel by viewModels()

```
property 선언과 함수 선언 사이에는 빈 줄을 하나 넣는 관습을 선호하는 개발자들이 많습니다.
단, "반드시"는 아니에요.